README.md

```diff
@@ -49,6 +49,22 @@ The module exposes a number of endpoints to provide a composite object that link
 * queryParameters supported: No
 * Permissions required?: No
 
+#### `/bl-users/by-id/<id>/open-transactions`
+* Description: Return an object listing number of open transactions that are associated to the user referenced by the user's id
+* Supported operations: GET
+* queryParameters supported: no
+* Permissions required?: Yes
+
+#### `/bl-users/by-username/<username>/open-transactions`
+* Description: Return an object listing number of open transactions that are associated to the user referenced by the user's username
+* Supported operations: GET
+* Permissions required?: Yes
```

Comment:
Why are these endpoints needed?

Is the intention that they are used by the client prior to attempting deletion or after a failed deletion to present more information (rather than including it in the response of the failed deletion)?

Reply:
The intention is that they are used prior to attempting deletion, e.g. via a campus identity management system when a user is leaving the university. The detailed use case is explained in [UXPROD-2904](https://issues.folio.org/browse/UXPROD-2904).

For a failed deletion additional information (an overview about open transactions) is already presented.

Reply:
Thanks for explaining that.

Personally, the name `transactions` makes me a little nervous, it seems a very general term to have defined. I'm aware that came from the PO asking for this change.